FrontEnd/components/Sidebar/Left/index.vue

```diff
@@ -121,19 +121,22 @@ import {
   DocumentTextIcon,
   PencilIcon,
   LogoutIcon
-} from "@heroicons/vue/solid"; // for solid icons
+} from "@heroicons/vue/solid"; 
 import { computed } from "vue";
-import { useUserStore } from "~/stores/useUserStore";
+import { useUserStore } from "~/stores/user";
+import { usePostStore} from "~/stores/post";
+
 
 const userStore = useUserStore();
+const postStore = usePostStore()
 const colorMode = useColorMode();
 
-const userId = computed(() => {
-  return userStore.userInfo.id;
-});
-const user = computed(() => {
-  return userStore.userInfo.name;
-});
+const userId = userStore.userId
+const user =  userStore.name
+
```

Comment:
line 134 and 135 store values ain't reactive.
Hint: check `storeToRefs` in pinia